animation_nodes/nodes/rotation/c_utils.pyx

```diff
@@ -121,3 +127,65 @@ def randomQuaternionList(int seed, int amount):
         quaternionNormalize_InPlace(result.data + i)
 
     return result
+
+def quaternionListToMatrixList(QuaternionList qs):
+    cdef Py_ssize_t i
+    cdef long amount = qs.length
+    cdef Matrix4x4List ms = Matrix4x4List(length = amount)
+
+    for i in range(amount):
+        quaternionToMatrix4(ms.data + i, qs.data + i)
+    return ms
+
+def eulerListToQuaternionList(EulerList es):
+    cdef Py_ssize_t i
+    cdef long amount = es.length
+    cdef QuaternionList qs = QuaternionList(length = amount)
+
+    for i in range(amount):
+        euler3ToQuaternion(qs.data + i, es.data + i)
+        quaternionNormalize_InPlace(qs.data + i)
```

Comment:
Are the normalize calls here necessary? If they are, normalization should be done inside the `euler3ToQuaternion` function.